common/src/config.rs

```diff
@@ -0,0 +1,125 @@
+use serde::Deserialize;
+use serde_json_core::de;
+
+#[derive(Deserialize)]
+pub struct BootloaderConfigFile {
```

Comment:
If we expose this in the API (see below), we should probably rename this to something else (e.g. `BootConfig`).